scipy/sparse/linalg/_dsolve/tests/test_linsolve.py

```diff
@@ -773,30 +773,49 @@ def test_input_types(self):
     @pytest.mark.slow
     @pytest.mark.timeout(120)  # prerelease_deps_coverage_64bit_blas job
```

Comment:
```suggestion
    @pytest.mark.timeout(120)  # prerelease_deps_coverage_64bit_blas job
```

Is this timeout still needed with the speedup this PR brings? Related: [#slow tests](https://github.com/scipy/scipy/issues/16494)

To find out, tests need to be approved by a maintainer again.

Reply:
The timeout could also be caused by some BLAS issues, but let's hope for the best. CI will tell us.